src/main/resources/view/PersonInfo.fxml

```diff
@@ -31,6 +31,9 @@
             <Label fx:id="email" styleClass="info_small_label" text="\$email" />
             <Label fx:id="phone" styleClass="info_small_label" text="\$phone" />
             <Label fx:id="address" styleClass="info_small_label" text="\$address" />
+            <Label fx:id="totalLeave" styleClass="info_small_label" text="\$totalLeave" />
+            <Label fx:id="leaveLeft" styleClass="info_small_label" text="\$leaveLeft" />
+
```

Comment:
Eh minor nit, but can delete this empty line if it's not doing anything?

Reply:
Yes Sir!